### PR TITLE
refactor: adapt to develop branch

### PR DESCRIPTION
**Pull request by 3Hren**

### CODE REVIEW

File: node/include/cocaine/detail/service/node/dispatch/client.hpp [hunk 1]
```diff
@@ -66,10 +66,6 @@ class client_rpc_dispatch_t:
     /// The client has been disconnected without closing its opened channels.
     ///
     /// In this case we should call close callback to prevend resource leak.
-    virtual
-    void
-    discard(const std::error_code& ec) const;
-
     void
     discard(const std::error_code& ec);
```
Comment (antmat) [inline node/include/cocaine/detail/service/node/dispatch/client.hpp]: override

File: node/include/cocaine/detail/service/node/dispatch/worker.hpp [hunk 2]
```diff
@@ -44,10 +44,6 @@ class worker_rpc_dispatch_t:
     /// The worker has been disconnected without closing its opened channels.
     ///
     /// In this case we should notify all users about the failure.
-    virtual
-    void
-    discard(const std::error_code& ec) const;
-
     void
     discard(const std::error_code& ec);
```
Comment (antmat) [inline node/include/cocaine/detail/service/node/dispatch/worker.hpp]: override

**Review by antmat — APPROVED** [review]
such much const, wow!

**Comment by 3Hren** [comment]
Fixed!

**Pull request merged**